images/cassandra-medusa/tests/medusa-install.sh

```diff
@@ -87,7 +51,7 @@ spec:
         size: 1
         storageConfig:
           cassandraDataVolumeClaimSpec:
-            storageClassName: standard
+            storageClassName: local-path
```

Comment:
the test harness seems to use local-path as the default sc name, looks like this was missed in previous update